test/tests.py

```diff
@@ -65,8 +65,7 @@ def active_test(fullname):
 bat_sfx = batch_suffix()
 exe_sfx = '.exe' if is_windows() else ""
 
-mini_python = [join(simics_base_path(), host_type(), "bin",
-                    "mini-python" + exe_sfx)]
+mini_python = [join(simics_repo_root(), "bin", f"mini-python{batch_suffix()}")]
```

Comment:
simics_repo_root is only available when running from the  base repo, and we want the test to be runnable from a normal project. So better to use `testparams.project_path` here. Also, we should check if this change gives a significant performance penalty on Windows; if so, we should consider adding something to t126 that makes it use mini-python.exe directly.